grid2op/Environment/MultiMixEnv.py

```diff
@@ -95,6 +95,42 @@ def __next__(self):
     def __getattr__(self, name):
         return getattr(self.current_env, name)
 
+    def keys(self):
+        for mix in self.mix_envs:
+            yield mix.name
```

Comment:
The name of the envs in "mix_envs" are the one of their respective folder right ? 
Also, that makes me think that, if a backend is passe as attribute of the constructor, the actual implementation will do weird stuff:
two "mix_envs" will have the same backend instance :fearful: